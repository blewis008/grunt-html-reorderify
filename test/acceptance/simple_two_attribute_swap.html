<html>
	<head>
		<title>Title</title>
	</head>
	<body>
		<div class="test-class" id="div1">
			<h1>Header</h1>
			<p>paragraphing</p>
		</div>
	</body>
</html>