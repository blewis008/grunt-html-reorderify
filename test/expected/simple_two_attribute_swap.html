<html>
	<head>
		<title>Title</title>
	</head>
	<body>
		<div id="div1" class="test-class">
			<h1>Header</h1>
			<p>paragraphing</p>
		</div>
	</body>
</html>